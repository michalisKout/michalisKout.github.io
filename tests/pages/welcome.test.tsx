import { render } from '@testing-library/react';
import Welcome from '../../pages/welcome';
import mockData from '__mocks__/userMock';

describe('Welcome page', () => {
  it('renders Welcome page', () => {
    const { asFragment } = render(<Welcome data={mockData} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
