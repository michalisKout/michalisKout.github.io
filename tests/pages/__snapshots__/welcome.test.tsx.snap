// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome page renders Welcome page 1`] = `
<DocumentFragment>
  <main
    class=""
  >
    <section
      class="main--section info--container pt-20 pb-20"
    >
      <div
        class="info--wrapper"
      >
        <div
          class="lg:w-1/2"
        >
          <div
            class="info--image"
            style="background-image: url(url);"
          />
        </div>
        <div
          class="info--content--wrapper"
        >
          <h2
            class="info--title"
          >
            Who I Am
            <span
              class="info--tag"
            >
               *
            </span>
          </h2>
          <p
            class="info--content"
          />
          <p>
            test value
          </p>
          <p />
        </div>
      </div>
    </section>
    <section
      class="secondary--section"
    >
      <div
        class="flex justify-center pt-20"
      >
        <h1
          class="user--title"
        >
          Some of my projects
        </h1>
      </div>
      <div
        class="projects--container"
      >
        <article
          class="project--container"
        >
          <div
            class="project--image-wrapper"
          >
            <div
              class="project--image"
              style="width: 350px;"
            />
          </div>
          <div
            class="project--title-wrapper"
          >
            <h3
              class="project--title capitalize"
            >
              title
            </h3>
            <div
              class="project--info-wrapper"
            >
              <span
                class="font-bold text-gray-800 dark:text-gray-200 uppercase text-xs max-w-sm"
              >
                stack
              </span>
              <a
                class="project--link"
                href="link"
              >
                github
              </a>
            </div>
          </div>
        </article>
        <article
          class="project--container"
        >
          <div
            class="project--image-wrapper"
          >
            <div
              class="project--image"
              style="width: 350px;"
            />
          </div>
          <div
            class="project--title-wrapper"
          >
            <h3
              class="project--title capitalize"
            >
              title
            </h3>
            <div
              class="project--info-wrapper"
            >
              <span
                class="font-bold text-gray-800 dark:text-gray-200 uppercase text-xs max-w-sm"
              >
                stack
              </span>
              <a
                class="project--link"
                href="link"
              >
                github
              </a>
            </div>
          </div>
        </article>
        <article
          class="project--container"
        >
          <div
            class="project--image-wrapper"
          >
            <div
              class="project--image"
              style="width: 350px;"
            />
          </div>
          <div
            class="project--title-wrapper"
          >
            <h3
              class="project--title capitalize"
            >
              title
            </h3>
            <div
              class="project--info-wrapper"
            >
              <span
                class="font-bold text-gray-800 dark:text-gray-200 uppercase text-xs max-w-sm"
              >
                stack
              </span>
              <a
                class="project--link"
                href="link"
              >
                github
              </a>
            </div>
          </div>
        </article>
      </div>
    </section>
    <section
      class="main--section pt-20 pb-20"
    >
      <div
        class="user--inner-container"
      >
        <div
          class="user--inner-wrapper"
        >
          <div
            class="lg:w-1/2"
          >
            <h1
              class="user--title"
            >
              Current position
            </h1>
            <section
              class="user--content"
            >
              <p>
                test value
              </p>
            </section>
            <div
              class="user--social-medial--container"
            >
              <a
                aria-label="Twitter"
                class="mx-2"
                href="#"
              >
                <span
                  class="icon-svg"
                >
                  <test />
                </span>
              </a>
              <a
                aria-label="Facebook"
                class="mx-2"
                href="#"
              >
                <span
                  class="icon-svg"
                >
                  <test />
                </span>
              </a>
              <a
                aria-label="Linkden"
                class="mx-2"
                href="#"
              >
                <span
                  class="icon-svg"
                >
                  <test />
                </span>
              </a>
              <a
                aria-label="Github"
                class="mx-2"
                href="#"
              >
                <span
                  class="icon-svg"
                >
                  <test />
                </span>
              </a>
            </div>
          </div>
          <div
            class="user--image-container"
          >
            <div
              class="user--image-wrapper"
            >
              <div
                class="max-w-lg"
              >
                <img
                  alt=""
                  class="user--image"
                  src="image url"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <footer
      class="footer"
    >
      <div
        class="container py-6"
      >
        <hr
          class="breaking-line"
        />
        <div
          class="footer--menu"
        >
          <div>
            <a
              class="link-primary"
              href="#"
            >
              michalis.koutridis@gmail.com
            </a>
          </div>
          <div
            class="footer--menu-wrapper"
          >
            Made with 
            <a
              href="https://nextjs.org/"
            >
               next js 
            </a>
            ,
            <a
              href="https://www.typescriptlang.org/"
            >
               typescript 
            </a>
            ,
            <a
              href="https://tailwindcss.com/"
            >
               tailwindcss 
            </a>
            ,
            <a
              href="https://www.contentful.com/"
            >
               contentful
            </a>
             and
            <a
              href="https://merakiui.com/"
            >
               merakiui 
            </a>
            .
          </div>
        </div>
      </div>
    </footer>
  </main>
</DocumentFragment>
`;
